### ISSUE TITLE
Feature request: Explorer backend fields

### ISSUE THREAD

**Issue by evgenykuzyakov** [near/near-explorer#184]
## Story

As a dev who uses Explorer CLI, I want to be able to find out how many accounts have a given contract installed so I can collect a statistics on the number of users in metanear.

## Missing Fields

There are a few fields missing to accomplish this query one way or another: 
- account.code_hash - current code hash deployed on an account
- transaction.actions - DeployContract doesn't have code hash either.

Unrelated fields I was interested in:
- account.block_hash or block_timestamp - when the account was created. I have to join with transactions table and then with blocks table
- transaction.block_hash or block_timestamp - when the transaction was processed. I have to join with the blocks table.
- maybe split transaction.actions into an indexed actions table, or add some structure for easy querying of transactions. Right now I have to use `actions like "%FunctionCall%"`.

**Comment by frol** [comment]
Thank you for giving CLI a try!

`DeployContract.code_hash` is not exposed (I am not even sure if there is such a thing). `account.code_hash` gets updated on every `DeployContract`, which will essentially require us to re-query the account RPC and maintain the latest change (which is tricky as we do concurrent queries to RPC to sync the network state). We should be able to track it properly in the next-generation Explorer backend.

Currently, I can only recommend getting the list of accounts (which is also incomplete since we do not take genesis into account yet #185), and query every single one of them with nearlib.

> * account.block_hash or block_timestamp - when the account was created. I have to join with transactions table and then with blocks table
> * transaction.block_hash or block_timestamp - when the transaction was processed. I have to join with the blocks table.

`block_timestamp` is the `timestamp`. JOIN also sounds fine to me. I would also consider removing the `timestamp` and use JOIN there as well, I failed to catch it on the review.

> * maybe split transaction.actions into an indexed actions table, or add some structure for easy querying of transactions. Right now I have to use actions like "%FunctionCall%".

Yes, this is definitely on the radar (there are some details in #185). This is legacy left from the times when we did not have actions and every transaction was an action.

In conclusion, there are no actionable items at the moment, but we want to keep track of such requests to better design the next-generation indexing backend.

**Comment by frol** [comment]
@evgenykuzyakov All the "unrelated fields" are now available (read my updated comment above). `code_hash` is still not indexed, though.

**Comment by frol** [comment]
Overall, Explorer CLI should be now deprecated and Indexer for Explorer has now all the data in Postgres database, so we can provide access for internal needs through Jupyter Hub ask @vgrichina for details if necessary. The data structure is visualized here: https://github.com/near/near-indexer-for-explorer#database-structure
